packages/client/lib/client/RESP2/decoder.ts

```diff
@@ -0,0 +1,229 @@
+import { ErrorReply } from '../../errors';
+import { Composer } from './composers/interface';
+import BufferComposer from './composers/buffer';
+import StringComposer from './composers/string';
+
+enum Types {
+    SIMPLE_STRING = 43, // +
```

Comment:
Good commenting on the actual ascii values.
IMHO a link to the RESP2 spec for the reader would be nice.